.ci/vfio_jenkins_job_build.sh

```diff
@@ -249,45 +249,22 @@ install_dependencies() {
 			# cloud image dependencies
 			deps=(xorriso curl qemu-utils openssh-client)
 
-			# QEMU dependencies
-			deps+=(libcap-dev libattr1-dev libcap-ng-dev librbd-dev gcc pkg-config libglib2.0-dev libpixman-1-dev psmisc ninja-build)
-
 			sudo apt-get update
 			sudo apt-get install -y ${deps[@]}
+			sudo apt-get install --reinstall -y qemu-system-x86
 			;;
-		fedora|centos|rhel)
+		fedora)
 			# cloud image dependencies
 			deps=(xorriso curl qemu-img openssh)
 
-			# QEMU dependencies
-			deps+=(libcap-devel libattr-devel libcap-ng-devel librbd-devel gcc glib2-devel pixman-devel psmisc ninja-build)
-
-			sudo dnf install -y ${deps[@]}
+			sudo dnf install -y ${deps[@]} qemu-system-x86-core
+			sudo dnf reinstall -y qemu-system-x86-core
```

Comment:
Late to the party, but we should rework this slightly in the next PR to remove the hard-coded architecture using the usual `arch=$(uname -m)` idiom. Or better, call https://github.com/kata-containers/tests/blob/main/.ci/kata-arch.sh as that makes life easier for some arches.